File: tasks-api-java/src/main/java/br/com/darlan/tasks/dto/TaskDTO.java
package br.com.darlan.tasks.dto;

import java.time.LocalDate;
import java.time.LocalTime;

import lombok.AllArgsConstructor;
import lombok.NoArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;
import org.springframework.hateoas.RepresentationModel;


@EqualsAndHashCode(callSuper = true)
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class TaskDTO extends RepresentationModel<TaskDTO> {
    Long idTask;
    String taskName;
    String taskDescription;
    Boolean uniqueExecution;
    LocalDate startDate;
    LocalTime startTime;
    LocalDate endDate;
    LocalTime endTime;
    String address;
    String host;
}

File: kotlin/tasks-api-kotlin/src/main/kotlin/br/com/darlan/tasks/dto/TaskDTO.kt
package br.com.darlan.tasks.dto

import java.time.LocalDate
import java.time.LocalTime

data class TaskDTO(var idTask: Long,
                   var taskName: String,
                   var taskDescription: String,
                   var uniqueExecution: Boolean,
                   var startDate: LocalDate,
                   var startTime: LocalTime,
                   var endDate: LocalDate,
                   var endTime: LocalTime,
                   var address: String,
                   var host: String)
